config/puma.rb

```diff
@@ -17,7 +17,7 @@
 environment ENV.fetch("RAILS_ENV") { "development" }
 
 # Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
+# pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
```

Comment:
I couldn't see an obvious reason for this change - could you explain why, and perhaps encapsulate it in its own commit?

Reply:
Yep, sorry, @kevindew wrote a perfectly good explanation in his fork of this branch - I will cherry-pick his commit into this branch
> Setting this pidfile requires a tmp/pids directory to be created
> otherwise puma will error. This is a pain point for us as in CI puma is
> used to run the JavaScript tests.
> 
> An alternate fix for this would be to use a git .keep in tmp/pids but
> the convention in existing GOV.UK apps is not to set this config.